src/crawlee/crawlers/_playwright/_playwright_crawler.py

```diff
@@ -210,6 +220,25 @@ async def _open_page(
                 await hook(pre_navigation_context)
         yield pre_navigation_context
 
+    def _prepare_request_handler(
+        self,
+        method: HttpMethod = 'GET',
+        headers: HttpHeaders | dict[str, str] | None = None,
+        payload: HttpPayload | None = None,
+    ) -> Callable:
+        """Create a route handler for Playwright to support non-GET methods with custom parameters.
```

Comment:
I'd mention that the purpose of the interception is to add custom headers and payload to the request.